src/runtime/pkg/katautils/config.go

```diff
@@ -426,7 +426,7 @@ func (h hypervisor) sharedFS() (string, error) {
 	supportedSharedFS := []string{config.Virtio9P, config.VirtioFS, config.VirtioFSNydus}
 
 	if h.SharedFS == "" {
-		return config.Virtio9P, nil
+		return config.VirtioFS, nil
```

Comment:
Why changes to virtiofs? This is a big change that is worth writing to changelog/release notes.

Reply:
I remember kata has set default  to `virtiofsd` before? I think it need fix so I updated.

Reply:
I'm sorry for that, truely the default is virtiofs: https://github.com/kata-containers/kata-containers/blob/be2e90469a48287091161d80e01fe473c110d7e7/src/runtime/Makefile#L166

Please ignore this.